db/db_local/backup.go

```diff
@@ -122,6 +127,8 @@ func stopDbByCmd(ctx context.Context, cmd *exec.Cmd) error {
 	return doThreeStepPostgresExit(ctx, p)
 }
 
+// needsBackup returns true if it finds any directory in `$STEAMPIPE_INSTALL_DIR/db` other than the ones it expects
```

Comment:
this comment leaves me more confused :) 

too many specifics and not enough intention

why does it return true if it finds other folders? When would this happen? Who uses this function?


``` // needsBackup returns whether the database needs backing up
// it called by/as part of....
// it checks whether.....
```